### PR TITLE
Sync should show warning not to share the code with anyone

### PR DESCRIPTION
**Pull request by deeppandya**
Addresses : https://github.com/brave/browser-android-tabs/issues/1475

### CODE REVIEW

File: chrome/android/java/strings/strings.xml [hunk 1]
```diff
@@ -109,7 +109,7 @@ Saved</string>
     <string name="brave_sync_btn_laptop">Add a Computer</string>
     <string name="brave_sync_scan_sync_code">Scan this sync code</string>
     <string name="brave_sync_add_mobile_device_text_part_1">Using a second phone or tablet, navigate to Brave Settings > Sync > Scan.</string>
-    <string name="brave_sync_add_mobile_device_text_part_2">Capture the QR code (above) with your second device, or enter code words if no camera is available.</string>
+    <string name="brave_sync_add_mobile_device_text_part_2">On your mobile device, navigate to Brave Sync in the Settings panel and click the button "Scan Sync Code". Use your camera to scan the QR Code below.</string>
```
Comment (samartnik) [inline chrome/android/java/strings/strings.xml]: Here we need to add string with new `name` as Transifex won't detect that this string should be translated.

File: chrome/android/java/strings/strings.xml [hunk 2]
```diff
@@ -258,7 +258,7 @@ content creators you love.</string>
     <string name="reset_brave_rewards_error_title">Reset Error</string>
     <string name="reset_brave_rewards_error_description">There was an error resetting Brave Rewards. Please relaunch Brave and try again</string>
 
-    <string name="brave_sync_add_laptop_text_part_2_new">Enter the sync chain code words shown below. Treat this code like a password. If someone gets ahold of it, they can read and modify your synced data.</string>
+    <string name="brave_sync_add_laptop_text_part_2_new">On your target computer, navigate to Brave Sync in settings and click the button "I have a Sync Code".\nEnter the sync chain code words shown below.\nTreat this code like a password. If someone gets hold of it, they can read and modify your synced data.</string>
```
Comment (samartnik) [inline chrome/android/java/strings/strings.xml]: Same here. We need string with new `name`.

File: chrome/android/java/strings/strings.xml (hunk 1 above)
Comment (deeppandya) [reply chrome/android/java/strings/strings.xml]: @samartnik then i think i should remove the text, we won't be using. correct me if i am wrong

Comment (samartnik) [reply chrome/android/java/strings/strings.xml]: @anthonypkeane is there any chance that we might use the old text? It's just if we delete it, next time, if we want to re-use it, we will have to re-translate it.

Comment (samartnik) [reply chrome/android/java/strings/strings.xml]: @deeppandya sorry for misleading, I tried it today, and turned out that different name is not required for Transifex. Current changes are ok as is.

Comment (deeppandya) [reply chrome/android/java/strings/strings.xml]: @samartnik Ok cool. so no changes required for this PR. right ?

Comment (samartnik) [reply chrome/android/java/strings/strings.xml]: yep, lgtm

**Pull request merged**